### PR TITLE
Rewrite Seqtk sample (single-end and paired-end) processes to Python

### PR DESCRIPTION
**Pull request by OtonicarJan**
## Checklist

* [ x] Update CHANGELOG.rst for each commit separately:
  * Pay attention to write entries under the "Unreleased" section.
  * Mark all breaking changes as "**BACKWARD INCOMPATIBLE:**" and put them
    before non-breaking changes.
  * If a commit modifies a feature listed under "Unreleased" section,
    it might be sufficient to modify the existing CHANGELOG entry from previous
    commit(s).
* [ x] Bump the process version:
  * **MAJOR version (first number)**: Backward incompatible changes (changes
    that break the api/interface). Examples: renaming the input/output, adding
    mandatory input, removing input/output...
  * **MINOR version (middle number)**: add functionality or changes in a
    backwards-compatible manner. Examples: add output field, add non-mandatory
    input parameter, use a different tool that produces same results...
  * **PATCH version (last number)**: changes/bug fixes that do not affect
    the api/interface. Examples: typo fix, change/add warning messages...
* [ x] All inputs are used in process.
* [ x] All output fields have a value assigned to them.

### CODE REVIEW

File: resolwe_bio/processes/support_processors/seqtk.py [hunk 1]
```diff
@@ -0,0 +1,324 @@
...
+        )
+        fastqc_archive = ListField(
+            FileField(),
+            label="Download FastQC archive",
+        )
+
+    def run(self, inputs, outputs):
+        """Run analysis."""
+
+        if inputs.advanced.fraction and not 0 < inputs.advanced.fraction < 1.0:
```
Comment (jkokosar) [inline resolwe_bio/processes/support_processors/seqtk.py:109]: Please check if this should be `0 < inputs.advanced.fraction <= 1` instead

File: resolwe_bio/processes/support_processors/seqtk.py [hunk 2]
```diff
@@ -0,0 +1,324 @@
...
+        )
+        fastqc_archive2 = ListField(
+            FileField(),
+            label="Download mate 2 FastQC archive",
+        )
+
+    def run(self, inputs, outputs):
+        """Run analysis."""
+
+        if inputs.advanced.fraction and not 0 < inputs.advanced.fraction < 1.0:
```
Comment (jkokosar) [inline resolwe_bio/processes/support_processors/seqtk.py:255]: See above

**Review by jkokosar — APPROVED** [review]
A small comment, otherwise looks good!

File: docs/CHANGELOG.rst [hunk 3]
```diff
@@ -38,6 +38,8 @@ Changed
   ``workflow-cutadapt-star-fc-quant-single``,
   ``workflow-cutadapt-star-fc-quant-wo-depletion-single``
 - Replace ANONYMOUS_USER_ID with ANONYMOUS_USER_NAME in settings.py
+- Rewrite processes ``seqtk-sample-single`` and ``seqtk-sample-paired``
```
Comment (jkokosar) [inline docs/CHANGELOG.rst:41]: Move the changelog entry to the unreleased section when rebasing

File: resolwe_bio/processes/support_processors/seqtk.py (hunk 1 above)
Comment (AnzeLovse) [reply resolwe_bio/processes/support_processors/seqtk.py:109]: FloatFields also support the range attribute that could replace this if sentence. 
```
fraction = FloatField(
    label="Fraction",
    required=False,
    range=[0.0, 1.0],
    description=...,
)
```

**Pull request merged**